docs/manual/working/scalaGuide/main/json/ScalaJson.md

```diff
@@ -1,7 +1,7 @@
 <!--- Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com> -->
 # JSON basics
 
-Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](api/libs/json/).
+Modern web applications often need to parse and generate data in the JSON (JavaScript Object Notation) format. Play supports this via its [JSON library](api/scala/play/api/libs/json).
```

Comment:
I'm not sure this link is what we want. I think we want to go to https://www.playframework.com/documentation/2.6.x/api/scala/play/api/libs/json/package.html, so the correct link should be `api/scala/play/api/libs/json/package.html`

Reply:
You are right. The link for 2.5.x version is https://www.playframework.com/documentation/2.5.x/api/scala/play/api/libs/json/package.html